xline/src/xline_utils/args.rs

```diff
@@ -0,0 +1,276 @@
+use std::{collections::HashMap, env, path::PathBuf, time::Duration};
+
+use anyhow::Result;
+use clap::Parser;
+use tokio::fs;
+use utils::{
+    config::{
+        default_batch_max_size, default_batch_timeout, default_candidate_timeout_ticks,
+        default_client_wait_synced_timeout, default_cmd_workers, default_compact_batch_size,
+        default_compact_sleep_interval, default_compact_timeout, default_follower_timeout_ticks,
+        default_gc_interval, default_heartbeat_interval, default_initial_retry_timeout,
+        default_log_entries_cap, default_log_level, default_max_retry_timeout,
+        default_propose_timeout, default_quota, default_range_retry_timeout, default_retry_count,
+        default_rotation, default_rpc_timeout, default_server_wait_synced_timeout,
+        default_sync_victims_interval, default_use_backoff, default_watch_progress_notify_interval,
+        AuthConfig, AutoCompactConfig, ClientConfig, ClusterConfig, CompactConfig,
+        CurpConfigBuilder, EngineConfig, InitialClusterState, LevelConfig, LogConfig,
+        RotationConfig, ServerTimeout, StorageConfig, TraceConfig, XlineServerConfig,
+    },
+    parse_batch_bytes, parse_duration, parse_log_level, parse_members, parse_rotation, parse_state,
+    ConfigFileError,
+};
+/// Command line arguments
+#[derive(Parser, Debug)]
+#[clap(author, version, about, long_about = None)]
+pub struct ServerArgs {
```

Comment:
Should this go to the `utils` crate?

Reply:
This struct only used by xline, it shouldn't move to utils